### PR TITLE
Mosquitto Addon: Add libcjson1 and libc-ares2 dependency for mosquitto_dynamic_security.so plugin/asynchronous name resolver dependencies

### PR DESCRIPTION
**Pull request by saschaludwig**
Enabling mosquitto_dynamic_security plugin prevents successful startup of Mosquitto addon.
Added libcjson1 package to apt install to prevent removal during clean up.

### CODE REVIEW

**Review by agners — APPROVED** [review]

**Comment by pvizeli** [comment]
Need version bump as well

**Comment by saschaludwig** [comment]
Hi Pascal, sorry for my question, but I'm not sure to what you are referring here.
> Need version bump as well

What exactly do you mean by that?

The patch/PR does not require a version bump. I tested the proposed changes, and they are successfully running in my environment since last week. But I'm happy to help if I can.

**Comment by agners** [comment]
> Or do you mean to bump the version here? 

Yes, in `config.yaml` of this add-on and then also add an entry to `CHANGELOG.md`.

File: mosquitto/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 6.2.1
+
+- Add dependencies for dynamic security plugin and asynchronous name resolver 
+- Prevent uninstallation of libcjson1 and libc-ares2 during cleanup
```
Comment (agners) [inline mosquitto/CHANGELOG.md:6]: This is essentially only change, so I'd suggest:

```suggestion
- Add explicit dependencies for dynamic security plugin and asynchronous name resolver 
```

**Review by agners — APPROVED** [review]

**Pull request merged**